index.js

```diff
@@ -14,6 +14,8 @@ function Argv (processArgs, cwd) {
   return argv
 }
 
+Argv.yargsa = require('./yargsa')
```

Comment:
My thinking is, let's call this `Argv.async`, at which point, rather than thinking of `yargsa` as the successor of yargs, we frame it as the import you should use if you're using the async version of the API.